commission/reports/report_settlement_templates.xml

```diff
@@ -17,6 +17,15 @@
             <div class="mt-5">
                 <div class="page">
                     <h2>Settlement</h2>
+                    <t t-if="show_doc_total">
+                        <div class="text-right">
+                            <strong>Global commission amount:</strong>
+                            <span
+                                t-esc="sum(docs.mapped('total'))"
+                                t-options="{'widget': 'monetary', 'display_currency': docs[0].currency_id}"
+                            />
+                        </div>
+                    </t>
```

Comment:
**issue**: Since this is the total for all pages being printed, I don't think it should be printed inside the `foreach` (line 95), as you only want this printed once, not once for every commission.

Reply:
Thank you for your observation. While it's true that the total is printed inside the for each loop, it actually maintains the same behavior as in the 14.0.